### PR TITLE
fix(web/RunsTable): Remove populate='metrics' from db query 

### LINKED ISSUE #47: "Error: No response was received from the server." for larger databases
For larger databases I get `Error: No response was received from the server.` (I suspect a timeout - sacredboard works fine). Can we remove the timeout?

### PR DESCRIPTION
**Pull request by matzek92**
to decrease memory usage.

fix #47

### CODE REVIEW

**Comment by vivekratnavel** [comment]
@matzek92 We cannot simply remove `populate=metrics` from the query. Without it, custom metric columns will not work. We can do performance optimization by selecting only the required metrics, but not simply remove `populate=metrics` and leave it like that

**Comment by matzek92** [comment]
@vivekratnavel I changed the request order to first request Omniboard.Columns to build the request for runs depending on the required metric columns. Are there any pitfalls to this change?

File: web/src/components/RunsTable/runsTable.js [hunk 1]
```diff
@@ -181,247 +181,267 @@ class RunsTable extends Component {
...
-            const host = data['host'];
-            delete data['host'];
-            data = {...data, 'hostname': host['hostname']};
-            const hostMap = this._getColumnNameMap(host, 'host');
-            columnNameMap = {...columnNameMap, ...hostMap};
+      if (metricColumnsData.length){
+        const metric_column_names = [];
+        metricColumnsData.forEach(column => {
+          metric_column_names.push(column.metric_name);
+        });
```
Comment (vivekratnavel) [inline web/src/components/RunsTable/runsTable.js]: This block can be replaced with
```
const metricColumnNames = metricColumnsData.map(column => column.metric_name);
```

Comment (matzek92) [reply web/src/components/RunsTable/runsTable.js]: I changed the code style. But I currently have no Idea how to change the tests in runsTable.test.js ... (Never worked with mockAxios).

Comment (vivekratnavel) [reply web/src/components/RunsTable/runsTable.js]: Let me try to help you with that when I find time. I will push commits to your branch - matzek92:fix-memory-issue

File: web/src/components/RunsTable/runsTable.js [hunk 2]
```diff
@@ -432,14 +434,6 @@ class RunsTable extends Component {
           errorMessage
         })
       });
```
Comment (vivekratnavel) [inline web/src/components/RunsTable/runsTable.js]: I think we will need to catch the errors and handle state accordingly here

**Comment by matzek92** [comment]
@vivekratnavel Thank you for your work :+1:  Now exploring several hundreds of my experiments is not a problem anymore.

**Comment by vivekratnavel** [comment]
@matzek92 Thank you for the PR! 👍

**Pull request merged**